<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSaveImage"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Tool</string>
    </property>
    <addaction name="actionGenerateTrainingData"/>
    <addaction name="actionMCMC"/>
   </widget>
   <widget class="QMenu" name="menuRendering">
    <property name="title">
     <string>Rendering</string>
    </property>
    <addaction name="actionRenderingBasic"/>
    <addaction name="actionRenderingLine"/>
    <addaction name="actionRenderingContour"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTool"/>
   <addaction name="menuRendering"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionGenerateTrainingData">
   <property name="text">
    <string>Generate Training Data</string>
   </property>
  </action>
  <action name="actionMCMC">
   <property name="text">
    <string>MCMC</string>
   </property>
  </action>
  <action name="actionSaveImage">
   <property name="text">
    <string>Save Image</string>
   </property>
  </action>
  <action name="actionRenderingBasic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Basic</string>
   </property>
  </action>
  <action name="actionRenderingLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Line</string>
   </property>
  </action>
  <action name="actionRenderingContour">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Contour</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
